velox/exec/Unnest.cpp

```diff
@@ -320,6 +308,50 @@ RowVectorPtr Unnest::generateOutput(const RowRange& range) {
       std::move(outputs));
 }
 
+template <bool isOuter>
```

Comment:
isOuter is not an expensive check, use the member field isOuter_ might be more precise. It can also help the PR simple.

Reply:
done